index.html

```diff
@@ -41,5 +41,142 @@ <h1 class="header-text">Hey there. I’m Raphael
+            <div class="bar"></div>
+            </div>
+            
+        </div>
+
+        <div class="cards-container">
+            <div class="card-item">
+                <div class="up-card"></div>
+                <div class="down-card">
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+
+            <div class="card-item">
+                
+                <div class="down-card">
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+
+            <div class="card-item">
+                
+                <div class="down-card">
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+            <div class="card-item">
+               
+                <div class="down-card">
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+
+            <div class="card-item">
+                
+                <div class="down-card">
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+            <div class="card-item">
+               
+                <div class="down-card">
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+        </div>
+
+
+        <section>
+            <div class="about-me">
+                <h3>About me</h3>
+                <div class="about-me-text">
+                 <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+                    </div>
+                    <div class="my-resume"><p>Get My Resume</p></div>
```

Comment:
This element is supposed to be a clickable item such that a potential recruiter can get a copy of your resume and as such, I suggest you use a button tag for this and also include the various button states.